types/LayoutType.ts

```diff
@@ -0,0 +1 @@
+export type LayoutType = "grid" | "list";
```

Comment:
it's just an idea for the future - you can consider to keep this info in `localStorage` and initialize `useLayoutType` from it 😁